personal-token.py

```diff
@@ -0,0 +1,14 @@
+import base64
+
+def get_personal_token():
+    user_name = ""
+    api_key = ""
```

Comment:
are the users supposed to edit this or should they pass these in as args?

Reply:
Moved them to commandline args